indico/web/client/js/react/components/principals/Search.jsx

```diff
@@ -141,6 +152,7 @@ const searchFactory = config => {
                         onRemove={() => {
                           onRemove(x);
                         }}
+                        avatarURL={x.avatarURL}
```

Comment:
I'd move it above the callbacks ;) looks cleaner IMO if the single-line props come first

Reply:
Fixed ;)